src/cascadia/CommandlineArgs/CommandlineArgs.vcxproj

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
copyright header?

Reply:
We haven't done that on _any_ of our other vcxproj's so I'm gonna say no?

Reply:
derp haha

Reply:
lol